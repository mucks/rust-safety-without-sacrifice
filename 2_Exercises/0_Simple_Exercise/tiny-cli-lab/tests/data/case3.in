# Edge-ish: zeros and single word
ADD 0 0
MUL 7 0
UPPER Go
REV wow
COUNT AAAA
