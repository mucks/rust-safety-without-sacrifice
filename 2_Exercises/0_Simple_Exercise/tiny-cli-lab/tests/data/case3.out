0
0
GO
wow
4
